src/App.tsx

```diff
@@ -2,6 +2,7 @@ import React from "react";
 import Homepage from "./pages/Homepage";
 import DisplayCards from "./pages/DisplayCards";
 import Preview from "./pages/Preview";
+import Replay from "./pages/Replay";
```

Comment:
would it be better to name this page as "GameComplete" or something nicer but in that context rather than Replay?
Since the screen does more than just provide the option to replay

Reply:
@PreetPatel yea maybe, small minor thing that can be changed later though.